pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -702,19 +696,35 @@ func (r *Builder) Tasks(vmRef ref.Ref) (list []*plan.Task, err error) {
 			vmRef.String())
 	}
 
-	for _, va := range vm.Volumes {
-		gb := int64(va.Size)
-		list = append(
-			list,
-			&plan.Task{
-				Name: fmt.Sprintf("%s-%s", r.Migration.Name, va.ID),
-				Progress: libitr.Progress{
-					Total: gb * 1024,
-				},
-				Annotations: map[string]string{
-					"unit": "MB",
-				},
-			})
+	taskMap := map[string]int64{}
+	imageID := workload.ImageID
+
+	if imageID != "" {
+		taskName := getVmSnapshotName(r.Context, workload.ID)
+		taskTotal := int64(0)
+		taskTotal = workload.Image.SizeBytes / 1024 / 1024
+		taskMap[taskName] = taskTotal
+	}
```

Comment:
I'm confused - so it didn't work properly for VMs with image(s) but we didn't see it because we argued import from openstack is limited to vms with volumes? (cc @bennyz)